mindsdb_evaluator/accuracy/__init__.py

```diff
@@ -7,6 +11,13 @@
     evaluate_cat_array_accuracy, \
     complementary_smape_array_accuracy
 
+
+__sklearn_accs__ = [acc for acc in __sklearn_accs__ if acc[0].lower() == acc[0]]
+
+sk_metrics = importlib.import_module('sklearn.metrics')
```

Comment:
This too. Not sure if it is a good idea to add modules from external sources to `globals()` I know it is sklearn, but in the future other developers might look a this code and try the same with other libraries of unknown origin. 

Just my opinion here, the code should work just fine.